impl/src/testing.rs

```diff
@@ -24,6 +41,31 @@ dependencies = [
     "
 }
 
+pub fn named_toml_contents(name: &str, version: &str) -> String {
+  vec![
```

Comment:
Nit: [indoc](https://docs.rs/indoc/1.0.3/indoc/) can make this better with dtolnay awesome.

```rust
indoc::formatdoc! { r#"
    [package]
    name =  {name}
    version = {version}

    [lib]
    path = "not_a_lib.rs"
"#, name = name, version=version }
```

Reply:
updated